book/animations.md

```diff
@@ -2023,7 +2031,7 @@ To implement `local_to_absolute`, we first need a new `map` method on
 "contents" of the transform and outputs a rect in post-transform
 space. For example, if the transform was `translate(20px, 0px)` then
 the output of calling `map` on a rect would translate it by 20 pixels
-in the *x* direction.
+in the `x` direction.
```

Comment:
Is this right? I think the copyeditor often suggests replacing `*x*` with `$x$`, which I've largely been ignoring (for single letters it will look the same), but backticks for code font are only correct if we're referring to a variable; here we're referring to a coordinate direction.